### PR TITLE
Bump version to 2.3.0.snapshot4

### PR DESCRIPTION
**Pull request by suyograo**

### CODE REVIEW

File: Gemfile.jruby-1.9.lock [hunk 1]
```diff
@@ -184,362 +186,362 @@ GEM
...
-      logstash-core (>= 2.0.0.beta2, < 3.0.0)
-    logstash-mixin-http_client (2.2.1)
+      logstash-core-plugin-api (~> 1.0)
+    logstash-mixin-http_client (2.2.3)
       logstash-codec-plain
-      logstash-core (>= 2.0.0, < 3.0.0)
+      logstash-core-plugin-api (~> 1.0)
       manticore (>= 0.5.2, < 1.0.0)
     logstash-mixin-rabbitmq_connection (2.3.0-java)
       logstash-core (>= 2.0.0.beta2, < 3.0.0)
```
Comment (colinsurprenant) [inline Gemfile.jruby-1.9.lock]: I think this guy was not updated?

Comment (suyograo) [inline Gemfile.jruby-1.9.lock]: Fixed https://github.com/logstash-plugins/logstash-mixin-rabbitmq_connection/pull/17/files

Comment (colinsurprenant) [inline Gemfile.jruby-1.9.lock]: ok, now we need to update lock file....

**Comment by suyograo** [comment]
@colinsurprenant updated to use 2.3.1 of the mixin

**Comment by colinsurprenant** [comment]
LGTM

**Pull request merged**